File: C++/Estrutura repetitiva/dentro.cpp
// Biblioteca(s)
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    // Variável
    int dentro, fora, quant, i, num;

    dentro = 0;
    fora = 0;

    // Entrada de dados
    cout << "Quantos numeros voce vai digitar? ";
    cin >> quant;

    // Estrutura repetitiva
    for (i=1; i <= quant; i++) {
        // Entrada de dados
        cout << "Digite um numero: ";
        cin >> num;

        // Estrutura condicional
        if (num > 20 || num < 10) {
            fora = fora + 1;
        }
        else {
            dentro = dentro + 1;
        }
    }

    // Saída de dados
    cout << "De acordo com os dados digitados temos " << dentro << " dentro e " << fora << " fora.";

    return 0;
}

File: Java/Estrutura repetitiva/dentro.java
// Biblioteca
import java.util.Scanner;

public class dentro {

	public static void main(String[] args) {
		// Variável
		int dentro, fora, quant, i, num;
		
		dentro = 0;
		fora = 0;
		
		// Para facilitar a leitura
		Scanner sc = new Scanner(System.in);
		
		// Entrada de dados
		System.out.print("Quantos números você vai digitar? ");
		quant = sc.nextInt();
		
		// Estrutura repetitiva
		for (i = 1; i <= quant; i++) {
			// Entrada de dados
			System.out.print("Digite um número: ");
			num = sc.nextInt();
			
			// Estrutura condicional
			if (num > 20 || num < 10) {
				fora += 1;
			}
			else {
				dentro += 1;
			}
		}
		
		// Saída de dados
		System.out.print("Existem " + dentro + " dentro e " + fora + " fora.");
		
		sc.close(); // Boa prática de programação
	}
}